app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,50 @@
+class FeedbacksController < ApplicationController
+  FEEDBACK_TYPES = %w[freetext satisfaction].freeze
+
+  def create
+    if params[:type].nil? || !FEEDBACK_TYPES.include?(params[:type])
```

Comment:
Technically the first half of this if statement is redundant. If params[:type] is nil, then FEEDBACK_TYPES will not include it.